src/app-admin.js

```diff
@@ -0,0 +1,491 @@
+import express from 'express';
+import * as uuid from 'uuid';
+import { errorMiddleware } from './util/middlewares.js';
+import validateUser from './util/validate-user.js';
+import getAccountDb, { isAdmin } from './account-db.js';
+import config from './load-config.js';
+import bodyParser from 'body-parser';
+
+let app = express();
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(errorMiddleware);
+
+export { app as handlers };
+
+export const TOKEN_EXPIRATION_NEVER = -1;
+
+const sendErrorResponse = (res, status, reason, details) => {
+  res.status(status).send({
+    status: 'error',
+    reason,
+    details,
+  });
+};
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏personally - I'm not really a fan of this abstraction. It makes the code less readable.